examples/src/main/scala/Simple.scala

```diff
@@ -79,5 +80,6 @@ object SimpleMain extends App {
   finally {
     transport.shutdown()
     server()
+    pool.shutdown()
```

Comment:
I forge the executor then it makes some sense for me to be responsible for shutting it down. But I don't have strong feelings about it. In practice it seems unlikely that I would want to shut down the remotely stuff and then continue to use the executor for something else. But if some idiot handed you the default scala Futures pool and you shut it down behind their back, that could be kind of rude.